efficientdet/utils.py

```diff
@@ -254,9 +254,7 @@ def batch_norm_class(is_training, strategy=None):
   if is_training and strategy == 'tpu':
     return TpuBatchNormalization
   elif is_training and strategy == 'gpus':
-    # TODO(fsx950223): use SyncBatchNorm after TF bug is fixed (incorrect nccl
-    # all_reduce). See https://github.com/tensorflow/tensorflow/issues/41980
-    return BatchNormalization
+    return SyncBatchNormalization
```

Comment:
How about the speed of SyncBatchNormalization for multiple GPUs?

Reply:
It's slower about 40% than BatchNormalization for multiple GPUs. I believe it's acceptable.

Reply:
According to https://github.com/tensorflow/tensorflow/blob/9489702e35b16a40a1accf3b8b5ed557efae10c7/tensorflow/python/keras/layers/normalization_v2.py#L151.
Should I split replica_ctx.all_reduce?

Reply:
I did not quite understand these comments, but I think 40% slower is fine.